intermediate_source/model_parallel_tutorial.py

```diff
@@ -23,11 +23,33 @@
 of model parallel. It is up to the readers to apply the ideas to real-world
 applications.
 
-**Recommended Reading:**
-
--  https://pytorch.org/ For installation instructions
--  :doc:`/beginner/blitz/data_parallel_tutorial` Single-Machine Data Parallel
--  :doc:`/intermediate/ddp_tutorial` Combine Distributed Data Parallel and Model Parallel
+**Comparison between DataParallel and DistributedDataParallel**
+
+You may be wondering what the differences between
+`DataParallel <https://pytorch.org/docs/stable/nn.html#torch.nn.DataParallel>`_
+(covered in this tutorial) and
+`DistributedDataParallel <https://pytorch.org/docs/stable/_modules/torch/nn/parallel/distributed.html#DistributedDataParallel>`_
+(covered in the next tutorial) are. First, the similarity: both ``DataParallel``
+and ``DistributedDataParallel``` implement data parallel training.
+
+- The difference is that ``DataParallel`` is single-process multi-thread,
+  and only works on a single machine, while ``DistributedDataParallel`` is
+  multi-process and works for both single- and multi- machine training.
+  Thus, even for single machine training, ``DistributedDataParallel``
+  is expected to be faster than ``DataParallel``.
+
+Here's our guideline on when to use each:
+
+- If your **model** is too large to fit on one GPU, use model parallel -
+  that is, splitting your model across multiple GPUs as described
+  `here <https://pytorch.org/tutorials/beginner/blitz/data_parallel_tutorial.html>`_
```

Comment:
link is wrong. It points to `DataParallel` tutorial. Model parallel is covered in this tutorial.